aws-for-fluent-bit.yaml

```diff
@@ -0,0 +1,110 @@
+package:
+  name: aws-for-fluent-bit
+  version: 2.31.12.20230629
+  epoch: 0
+  description: AWS provides a Fluent Bit image with plugins for both CloudWatch Logs and Kinesis Data Firehose.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - glibc
+      - glibc-dev
+      - yaml-dev
+      - gcc
+      - make
+      - wget
+      - gnutar
+      - git
+      - openssl-dev
+      - cyrus-sasl-dev
+      - pkgconf
+      - dpkg
+      - systemd-dev
+      - zlib-dev
+      - flex
+      - bison
+      - autoconf
+      - automake
+      - go
+      - aws-flb-kinesis-compat
+      - aws-flb-firehose-compat
+      - aws-flb-cloudwatch-compat
+  environment:
+    FLB_VERSION: "1.9.10"
+    FLB_DOCKER_BRANCH: "1.8"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/aws/aws-for-fluent-bit
+      expected-commit: dfb7a7f63eadbe46a846cfbff05e57948154d39c
+      tag: v${{package.version}}
+
+  - runs: |
+      # GHSA-p782-xgp4-8hr8
+      go get -u golang.org/x/sys@v0.0.0-20220412211240-33da011f77ad
+
+      go mod tidy
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/fluent-bit/etc
```

Comment:
can we switch this to `"${{targets.destdir}}"/etc/fluent-bit` and add a compat subpackage?